src/app/features/home/home.page.ts

```diff
@@ -197,9 +198,17 @@ export class HomePage {
 
   // eslint-disable-next-line class-methods-use-this
   async openCaptureClub() {
-    return Browser.open({
-      url: 'https://captureclub.cc/',
-      toolbarColor: '#564dfc',
-    });
+    return this.diaBackendAuthService.token$
```

Comment:
```suggestion
    this.diaBackendAuthService.token$
```
The function now returns the subscription, but we'd almost never need to get the return value of a subscription. Better return void instead.